src/components/CreateNewPin.js

```diff
@@ -0,0 +1,71 @@
+import React, { useState, useEffect } from "react";
+import {
+  CreateNewPinWrapper,
+  PinDataUploadWrapper,
+  NewPinDataWrapper,
+  PinImageUploadWrapper,
+  NewPinDataTitle,
+  NewPinDataInput,
+  CreatePinButton,
+  UploadNewPinImageLabel,
+  UploadNewPinImageInput,
+} from "../styles/CreateNewPin.module";
+
+function CreateNewPin() {
+  const [pinName, setPinName] = useState();
+  const [pinDescription, setPinDescription] = useState();
+  const [pinLink, setPinLink] = useState();
+
+  const submitPinData = () => {
+    if (pinName && pinDescription && pinLink) {
+      console.log("pinName", pinName);
+      console.log("pinDescription", pinDescription);
+      console.log("pinLink", pinLink);
+    } else {
+      alert("please check if all fields are filled~");
+    }
+  };
```

Comment:
Use 'Early return' tip. Try not to use else as less as possible. You could implement like below:

if (!pinName || !pinDescription || !pinLink) {
    alert("please check if all fields are filled~");
}

// do what you want here